pkg/vfs/vfs.go

```diff
@@ -616,7 +617,8 @@ func normalizeDocPatch(data, patch *DocPatch, cdate time.Time) (*DocPatch, error
 	}
 
 	if patch.UpdatedAt.Before(cdate) {
-		return nil, ErrIllegalTime
```

Comment:
@Ljinod if Alice is using cozy-desktop and update the "todo.txt" file at 00:02 while being offline, and if she comes back online at 00:07, what is your expectations about the "todo.txt" file (its created_at and updated_at)?

For me, it should be created_at=00:00 and updated_at=00:02 for both.